bigbluebutton-html5/imports/ui/components/breakout-room/breakout-room/queries.ts

```diff
@@ -60,7 +60,7 @@ export const getBreakoutData = gql`
 
 export const getIfUserJoinedBreakoutRoom = gql`
   subscription getIdUserJoinedABreakout {
-    breakoutRoom_aggregate(where: {hasJoined: {_eq: true}}) {
+    breakoutRoom_aggregate(where: {isUserCurrentlyInRoom: {_eq: true}}) {
       aggregate {
         count
       }
```

Comment:
If you modify the information that a function returns, you should also rename the function and update the  subscription name to reflect the data it will provide.

`getIfUserJoinedBreakoutRoom` -> `getIsUserCurrentlyInBreakoutRoom`